.travis.yml

```diff
@@ -16,6 +16,8 @@ matrix:
       env: TOXENV=py38
     - python: 3.9-dev
       env: TOXENV=py39
+    - python: pypy3
+      env: TOXENV=pypy3 PYPY_VERSION=3.5-v7.0.0
```

Comment:
`PYPY_VERSION` is not something Travis understands. [We use it in Scrapy to manually install that version before the tests start](https://github.com/scrapy/scrapy/blob/master/.travis.yml#L49-L57).